examples/velocityGAN/functions.py

```diff
@@ -0,0 +1,248 @@
+from math import exp
+
+import matplotlib.pyplot as plt
+import numpy as np
+import paddle
+from matplotlib.colors import ListedColormap
+from paddle.vision.transforms import Compose
+
+rainbow_cmap = ListedColormap(np.load("rainbow256.npy"))
+
+
+def plot_velocity(output, target, path, vmin=None, vmax=None):
+    fig, ax = plt.subplots(1, 2, figsize=(11, 5))
+    if vmin is None or vmax is None:
+        vmax, vmin = np.max(target), np.min(target)
+    im = ax[0].matshow(output, cmap=rainbow_cmap, vmin=vmin, vmax=vmax)
+    ax[0].set_title("Prediction", y=1.08)
+    ax[1].matshow(target, cmap=rainbow_cmap, vmin=vmin, vmax=vmax)
+    ax[1].set_title("Ground Truth", y=1.08)
+
+    for axis in ax:
+        axis.set_xticks(range(0, 70, 10))
+        axis.set_xticklabels(range(0, 700, 100))
+        axis.set_yticks(range(0, 70, 10))
+        axis.set_yticklabels(range(0, 700, 100))
+
+        axis.set_ylabel("Depth (m)", fontsize=12)
+        axis.set_xlabel("Offset (m)", fontsize=12)
+
+    fig.colorbar(im, ax=ax, shrink=0.75, label="Velocity(m/s)")
+    plt.savefig(path, format="jpg")
+    plt.close("all")
+
+
+def create_transform(ctx, k):
+    log_data_min = log_transform(ctx["data_min"], k)
+    log_data_max = log_transform(ctx["data_max"], k)
+    transform_data = Compose(
+        [LogTransform(k), MinMaxNormalize(log_data_min, log_data_max)]
+    )
+    transform_label = Compose([MinMaxNormalize(ctx["label_min"], ctx["label_max"])])
+
+    return transform_data, transform_label
+
+
+def minmax_normalize(vid, vmin, vmax, scale=2):
+    vid -= vmin
+    vid /= vmax - vmin
+    return (vid - 0.5) * 2 if scale == 2 else vid
+
+
+class MinMaxNormalize(object):

```

Comment:
python3的类已经不需要手动从object继承，可以删除

```suggestion
class MinMaxNormalize:

```